src/wp-admin/includes/file.php

```diff
@@ -2557,3 +2625,32 @@ function wp_opcache_invalidate( $filepath, $force = false ) {
 
 	return false;
 }
+
+/**
+ * Invalidate OPcache of directory of files.
+ *
+ * @since 6.2.0
+ *
+ * @global WP_Filesystem_Base $wp_filesystem WordPress filesystem subclass.
+ *
+ * @param string|array $dir  The path to invalidate, or the results of ::dirlist().
```

Comment:
```suggestion
 * @param string|array $dir  The path to invalidate, or the results of WP_Filesystem_Base::dirlist().
```
Or similar to clarify that this is a method and not a general function (PHPUnit `@covers` tags would be `::dirlist()`, for example, and may cause confusion).

Possible alternative:

> The path to invalidate, or the results of the filesystem's dirlist() method.

What do you think?